test/scripts/topology_helper.py

```diff
@@ -61,3 +62,58 @@ def root(self) -> int:
         if self.__root is None:
             self.get()
         return self.__root
+
+
+if __name__ == "__main__":
+    """Command line option if this script is invoked directly
+
+    CLI Options that will print topology information if this script
+    is invoked directly
+    """
+    help = False
+    root = False
+    port = None
+    port_arg = False
+    baud = None
+    baud_arg = False
+
+    for arg in sys.argv:
+        if arg == "--help" or arg == "-h":
+            help = True
+            break
+        elif arg == "--root" or arg == "-r":
+            root = True
+        elif arg == "--port" or arg == "-p":
```

Comment:
Let's replace this with argparse: https://docs.python.org/3/library/argparse.html It generates the help text in a standard way automatically and handles more edge cases than we'll think of, plus it'll reduce this code a lot.